### PR TITLE
[3.2.0] Update Keymanager compatibility

### PR DESCRIPTION
**Pull request by Sarangan0219**

### CODE REVIEW

**Review by tharindu1st — APPROVED** [review]

File: en/docs/install-and-setup/setup/reference/product-compatibility.md [hunk 1]
```diff
@@ -66,11 +66,14 @@ The following is a list of other WSO2 products that is tested with WSO2 API Mana
...
 <tr class="even">
 <td>API-M-3.2.0 WUM update</td>
-<td>GA or WUM update for WSO2 IS-KM-5.10.0</td>
+<td>GA or WUM update for WSO2 IS-5.10.0</td>
 </tr>
 </tbody>
 </table>
+
+!!! note 
+    From API Manager 3.2.0 onwards, the prepackaged WSO2 Identity Server as a Key Manager is deprecated and WSO2 Identity Server should be used as the key Manager.
```
Comment (tharindu1st) [inline en/docs/install-and-setup/setup/reference/product-compatibility.md:79]: Theres no deprecation it was removed

**Review by tharindu1st — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/setup/reference/product-compatibility.md (hunk 1 above)
Comment (Sarangan0219) [reply en/docs/install-and-setup/setup/reference/product-compatibility.md]: https://github.com/wso2/docs-apim/pull/2157/commits/20b7bbc6fc82319a9a777cae67836e1280fd9383

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@Sarangan0219 , the removal of prepackaged WSO2 Identity Server as a Key Manager needs to be mentioned in the about this release page as well, because that this the common place that specifies what has changed in the release.

**Comment by Sarangan0219** [comment]
> @Sarangan0219 , the removal of prepackaged WSO2 Identity Server as a Key Manager needs to be mentioned in the about this release page as well, because that this the common place that specifies what has changed in the release.

ACK. Will add this change in that section.

**Comment by Mariangela** [comment]
I created https://github.com/wso2/docs-apim/issues/2197 to track the last comment

**Pull request merged**